src/amber/pipes/logger.cr

```diff
@@ -7,7 +7,7 @@ module Amber
 
       def initialize(@filter : Params = log_config.filter,
                      @skip : Params = log_config.skip,
-                     @context : Params = log_config.context)
+                     @context : Array(String) = log_config.context)
```

Comment:
Is this a problem for `filter` and `skip` as well? Maybe we need to do this for all the Params.  Is so, should we update the Params type instead?

Reply:
I uploaded the code I have, which should work, really, but it throws me this exception:

```
Unhandled exception: Expected String, not  at line 8, column 6 (YAML::ParseException)
  from /usr/share/crystal/src/yaml/nodes/nodes.cr:30:9 in 'raise'
  from /from_yaml.cr:34:7 in 'parse_scalar'
  from /usr/share/crystal/src/yaml/from_yaml.cr:56:3 in 'new'
  from /usr/share/crystal/src/yaml/from_yaml.cr:88:11 in 'new'
  from /usr/share/crystal/src/yaml/from_yaml.cr:140:3 in 'new'
  from src/amber/environment/settings.cr:49:5 in 'initialize'
  from src/amber/environment/settings.cr:49:5 in 'new'
  from /usr/share/crystal/src/yaml/from_yaml.cr:2:3 in 'from_yaml'
  from src/amber/environment/loader.cr:9:7 in 'settings'
  from src/amber.cr:24:3 in 'env='
  from spec/spec_helper.cr:14:1 in '__crystal_main'
  from /usr/share/crystal/src/crystal/main.cr:104:5 in 'main_user_code'
  from /usr/share/crystal/src/crystal/main.cr:93:7 in 'main'
  from /usr/share/crystal/src/crystal/main.cr:133:3 in 'main'
  from __libc_start_main
  from _start
  from ???
```

I am anable to figure out if this is a new error that surfaced with this, or an issue with this PR itself. Also, I am going on a vacations and will not be able to push this further after today, sorry: (. Next month I will be back.